ui/src/components/AddPoolModal.tsx

```diff
@@ -583,7 +686,7 @@ export function AddPoolModal({
               <div className={cn('my-6')}>
                 <ProgressBar
                   value={currentStep * (100 / totalSteps)}
-                  color="rose"
+                  color={currentStep === 3 ? (theme === 'light' ? 'blue' : 'teal') : 'rose'}
```

Comment:
Changing the color of the progress bar for the xGov step? Hmmm OK I'll allow it 😉